.travis.yml

```diff
@@ -262,8 +228,12 @@ before_script:
       export BUILD_TYPE=Release
     fi
   - mkdir build
-  # Use apt-installed cmake rather than travis-provided one (/usr/local/cmake-3.12.4/bin/cmake).
-  - /usr/bin/cmake -S . -B build -DCMAKE_BUILD_TYPE=$BUILD_TYPE -DENABLE_DIST_GALOIS=ON || exit 1
+  # Use apt-installed llvm-7-dev rather than travis-provided one which is
```

Comment:
Good catch. I wasn't aware that Travis provided one by default.

Reply:
I can add something in the comment here about the RTTI requirement?

This is actually the root reason why we can't use /usr/local/clang-7 on travis.

Reply:
IDK if we need to mention this in the build script. We should check that LLVM_ENABLE_RTTI is enabled after doing `find_package(llvm ...)` though. I have a patch to do this. I'll push it to this branch and you can confirm that it seems reasonable to you.

Reply:
Can't figure out how to push to this PR so I'll just submit a separate one.

Reply:
https://github.com/IntelligentSoftwareSystems/Galois/pull/122

Reply:
Hmm. I thought "allow edits from maintainers" would allow you to write to my branch. Oh well. I added comments to #122 and cherry-picked your changes here (with modifications).

If the only remaining issue is providing more visibility for RTTI errors. I can work on that here and ping you for another review when it is ready?